fl4health/clients/moon_client.py

```diff
@@ -106,8 +89,36 @@ def update_before_train(self, current_server_round: int) -> None:
         # Save the parameters of the global model
         self.global_model = self.clone_and_freeze_model(self.model)
 
+        if self.mkmmd_loss_weights and len(self.old_models_list) > 0:
+            self.set_optimized_betas(self.mkmmd_loss, self.old_models_list[-1], self.global_model)
+
         return super().update_before_train(current_server_round)
 
+    def set_optimized_betas(
+        self, mkmmd_loss: MkMmdLoss, old_model: torch.nn.Module, global_model: torch.nn.Module
+    ) -> None:
+        """Set the optimized betas for the MK-MMD loss."""
+        assert isinstance(old_model, MoonModel)
+        assert isinstance(global_model, MoonModel)
+
+        old_dist = []
```

Comment:
A similar comment to the FENDA client, where I think "_dist" might be a misleading suffix?